container_files/assets/www/doc/builtin/procedures/JSONImporter.md

```diff
@@ -0,0 +1,72 @@
+# JSON Import Procedure
+
+The JSON Import Procedure type is used to import a text file containing
+one JSON per line in a dataset.
+
+
+## Configuration
+
+![](%%config procedure import.json)
+
+## Example of data recording
+
+The following algorithm is used to record data:
+
+Each `(key, value)` pair will be recorded as the column name and cell value respectively.
+
+The line:
+
+    {"a": 5, "b": true}
+
+is recorded as:
+
+| *rowName* | *a* | *b* |
+|-----------|-----|-----|
+| row1 | 5 | true |
+
+If the value is an object, we apply the same logic recursively, adding an underscore
```

Comment:
Are we using period '.' in some other cases to flatten a nested object?  If so, we may want to use the same separator.